lib/linguist/languages.yml

```diff
@@ -3894,6 +3894,14 @@ Ring:
   tm_scope: source.ring
   ace_mode: text
   language_id: 431
+Riot:
+  type: programming
```

Comment:
This should be `markup`, not `programming`.

Reply:
Updated